README.md

```diff
@@ -154,6 +154,17 @@ $ dotenv list --format=json
 $ dotenv run -- python foo.py
 ```
 
+The CLI interface also supports reading from multiple `.env` files.
+
+```shell
+$ echo -e "a=1\nb=2\n" > .env1
+$ echo -e "b=20\nc=30\n" > .env2
+$ python -m dotenv -f .env1 -f .env2 list
+a=1
+b=20
+c=30
+```
```

Comment:
@duarte-pompeu Consider document the precedence/loading order values from the of the files. I didn't see a test but based on the this example it seems the `.env2` overrides the previous value from `.env1`.

Reply:
I agree: it's important to document this. I chose an overriding behavior, as my use case would be:

- have a big file with all values
- have smaller .env files with specific values to override in certain situations

I tried to improve it on 363aab5b3b228e670c09019b134570949faf5de3